src/domains/login/common/components/PasswordInput.jsx

```
@@ -34,7 +32,7 @@ const PasswordInput = ({ register, errors, watch }) => {
           />
           <div className='flex items-start'>
             {errors.passwordCheck && (
```

ESLint

[error] 'errors.passwordCheck' is missing in props validation

(react/prop-types)